modules/persistent-storage-csi-azure-disk-perf-plus-from-snapshots.adoc

```diff
@@ -0,0 +1,105 @@
+//
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-azure.adoc
+//
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-azure-disk-perf-plus-from-snapshots_{context}"]
+= Creating new disks from existing disks or snapshots with performance plus enabled
```

Comment:
That won't work with OCP, this is for users who directly interact with Azure

Reply:
Removed.